srcpkgs/metalog/template

```diff
@@ -1,19 +1,22 @@
 # Template file for 'metalog'
 pkgname=metalog
-version=3
-revision=6
+version=2018510
+revision=1
+wrksrc="$pkgname-$pkgname-$version"
 build_style=gnu-configure
-configure_args="--sbindir=/usr/bin"
-hostmakedepends="pkg-config"
+hostmakedepends="autoconf autoconf-archive automake pkg-config"
 makedepends="pcre-devel"
 conf_files="/etc/metalog.conf"
-short_desc="A modern replacement for syslogd and klogd"
+short_desc="Modern replacement for syslogd and klogd"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2"
```

Comment:
please use SPDX short-identifier

Reply:
I’ve looked, but can only find GPL-2, no headers in the source. Should it be GPL-2.0-only then?